pkg/client/client.go

```diff
@@ -412,4 +412,15 @@ func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, allAddresses,
 	} else {
 		fmt.Fprintf(w, "Proxy Status:\tNo managed proxy redirect\n")
 	}
+
+	if sr.Hubble != nil {
+		h := sr.Hubble
+		msg := h.Msg
+		if h.State == models.HubbleStatusStateOk && msg == "" && h.MaxFlows > 0 {
```

Comment:
if the status is not ok, what's wrong with printing the buffer contents?

grpc server failing, but buffer being populated is a valid data point i think

Reply:
The way the status is implemented right now, a non-empty `Msg` and us being able to obtain the buffer statistics are mutually exclusive. But I can change the client logic to be future proof in case this ever changes.